src/data/load_geometry.js

```diff
@@ -32,7 +32,7 @@ function preparePoint(point: Point, scale: number) {
 }
 
 // a subset of VectorTileGeometry
-type FeatureWithGeometry = {
+interface FeatureWithGeometry {
```

Comment:
A few changes like this in the PR — can you leave a comment on what's the difference here and why it fixes Flow complaining?

Reply:
In Flow [v0.153.0](https://github.com/facebook/flow/releases/tag/v0.153.0), objects are no longer valid supertypes of class instances, so after upgrading, it emits the `class-object-subtyping` error. In our case, several object types appear as supertypes of classes, i.e., `IVectorTileFeature` can no longer be a subtype of `FeatureWithGeometry`.

Interfaces can describe both instances and objects, unlike object types which can only describe objects. https://medium.com/flow-type/sound-typing-for-this-in-flow-d62db2af969e